### ISSUE TITLE
Need more sample like this! Thank-you 

### ISSUE THREAD

**Issue by mrshahzeb7** [Aashu-Dubey/React-Native-UI-Templates#1]
what about "Design course" and "My diary" it will be so much helpful if you make that one as well because right now it is showing a toast "comming soon"
Thankyou

**Comment by Aashu-Dubey** [comment]
Yes I'm working on those as well, Although can't tell you exactly when they will be available.

**Comment by mrshahzeb7** [comment]
I can help in design you can add the animation!
Can i help?
